packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text.cypress.ts

```diff
@@ -0,0 +1,314 @@
+            addMultiValueText(
+              {field, 'max-values-to-display': maxValuesToDisplay},
+              slot
+            )
+          )
+          .withCustomResponse((response) => {
+            response.results.forEach((result) => {
+              result.raw[field] = Object.keys(valuesAndCaptions);
+            });
+          })
+          .withFieldCaptions(field, valuesAndCaptions)
+          .withTranslation({'n-more': '{{value}} more'});
+      }
+
+      describe('with 2 values', () => {
+        const values = {
+          first: 'The first value',
+          second: 'The last value',
+        };
+        const localizedValues = Object.values(values);
+
+        describe('with max-values-to-display=1', () => {
+          beforeEach(() => {
+            prepareValuesWithMaximum(values, 1).init();
+          });
+
+          assertShouldRenderValues(localizedValues.slice(0, 1));
+
+          assertDisplaysXMoreLabel(1);
+        });
+      });
+
+      describe('with 3 values', () => {
+        const values = {
+          first: 'The first value',
+          second: 'Another value',
+          third: 'The last value',
+        };
+        const localizedValues = Object.values(values);
+
+        describe('with max-values-to-display=1', () => {
+          beforeEach(() => {
+            prepareValuesWithMaximum(values, 1).init();
+          });
+
+          assertShouldRenderValues(localizedValues.slice(0, 1));
+
+          assertDisplaysXMoreLabel(2);
+        });
+
+        describe('with max-values-to-display=2', () => {
+          beforeEach(() => {
+            prepareValuesWithMaximum(values, 2);
+          });
+
+          assertShouldRenderValues(localizedValues.slice(0, 1));
+
+          assertDisplaysXMoreLabel(2);
+        });
+      });
+
+      describe('with 4 values', () => {
+        const values = {
+          first: 'The first value',
+          second: 'Another value',
+          third: 'Almost the last value',
+          fourth: 'The last value',
+        };
+        const localizedValues = Object.values(values);
+
+        describe('with max-values-to-display=1', () => {
+          beforeEach(() => {
+            prepareValuesWithMaximum(values, 1).init();
+          });
+
+          assertShouldRenderValues(localizedValues.slice(0, 1));
+
+          assertDisplaysXMoreLabel(3);
+        });
+
+        describe('with max-values-to-display=2', () => {
+          beforeEach(() => {
+            prepareValuesWithMaximum(values, 2).init();
+          });
+
+          assertShouldRenderValues(localizedValues.slice(0, 2));
+
+          assertDisplaysXMoreLabel(2);
+        });
+
+        describe('with max-values-to-display=3', () => {
+          beforeEach(() => {
+            prepareValuesWithMaximum(values, 3).init();
+          });
+
+          assertShouldRenderValues(localizedValues.slice(0, 2));
+
+          assertDisplaysXMoreLabel(2);
+
+          it.skip('should pass accessibility tests', () => {
```

Comment:
if we skip because of issue, would be great if we add TODO + Jira number, so we can follow Jira and enable it later ;)